.gitattribute

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
The `.gitattribute` can be set system-wide within your machine, I am not sure it makes sense to add it to the repo.

Reply:
No problem, I've found some resource suggesting to add it tothe repo [[1](https://dev.to/deadlybyte/please-add-gitattributes-to-your-git-repository-1jld), [2](https://stackoverflow.com/a/15108707/3673430), [3](https://rehansaeed.com/gitattributes-best-practices/)] but I'm ok with whatever reduce the impact on other devs.